compiler/src/dotty/tools/dotc/transform/patmat/Space.scala

```diff
@@ -434,7 +434,7 @@ object SpaceEngine {
    *
    *  We cannot use type erasure here, as it would lose the constraints
    *  involving GADTs. For example, in the following code, type
-   *  erasure would loose the constraint that `x` and `y` must be
+   *  erasure would lose the constraint that `x` and `y` must be
```

Comment:
Interestingly, to lose a constraint is to loose the constraint.

Reply:
I did spend a few moments pondering if "loosen" might have been meant.